### PR TITLE
set up release notes

### PR DESCRIPTION
**Pull request by marcofilippi**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/release-notes/v13.1.1.md [hunk 1]
```diff
@@ -5,15 +5,7 @@ sidebar_label: v13.1.1
...
-_September 12th, 2024_
-
-:::info
-Mia-Platform Console v13.1.1 is **now in Preview** and will be generally available on September 27th.
-
-Console SaaS users can try out v13.1.1 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
-
-For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
-:::
+_September 27th, 2024_
```
Comment (albertotessarotto) [inline docs/release-notes/v13.1.1.md:8]:
```suggestion
_September 30th, 2024_

```

File: docs/release-notes/v13.2.0.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+* Added support for the mongo operator `size` in the `_q` parameter for fields of type `array`.
+
+##### Bug Fixes
+
+* Enabled bom stripping when importing CSV files
+* Changed CSV _escape_ character to align it with CSV field _quote_ character, as recommended by [RFC 4180](https://datatracker.ietf.org/doc/html/rfc4180)
+
+## How to update your Console
+
+For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.2-beta.0`.
```
Comment (danielemarostica) [inline docs/release-notes/v13.2.0.md:199]: the correct tag for v13.2.0 is v13.7.3-beta.X

```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.3-beta.0`.

```

Comment (albertotessarotto) [inline docs/release-notes/v13.2.0.md:199]: to change (maybe v13.7.3-beta.0?)

Comment (danielemarostica) [reply docs/release-notes/v13.2.0.md:199]: yes, updated with suggestion above :+1:

File: docs/release-notes/v13.2.0.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+* Added support for the mongo operator `size` in the `_q` parameter for fields of type `array`.
+
+##### Bug Fixes
+
+* Enabled bom stripping when importing CSV files
+* Changed CSV _escape_ character to align it with CSV field _quote_ character, as recommended by [RFC 4180](https://datatracker.ietf.org/doc/html/rfc4180)
+
+## How to update your Console
+
+For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.3-beta.2`.
```
Comment (danielemarostica) [inline docs/release-notes/v13.2.0.md:197]:
```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/infrastructure/self-hosted/installation-chart/100_how-to-upgrade.md) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v13.7.3-beta.3`.
```

**Review by albertotessarotto — APPROVED** [review]

**Review by jacoposte — APPROVED** [review]

**Pull request merged**